ambassador/ambassador/envoy/v2/v2cluster.py

```diff
@@ -35,11 +35,16 @@ def __init__(self, config: 'V2Config', cluster: IRCluster) -> None:
             else:
                 dns_lookup_family = 'V6_ONLY'
 
+        cluster_idle_timeout_ms = cluster.ir.ambassador_module.get('cluster_idle_timeout_ms', 30000)
```

Comment:
We might actually want a 60s default, which is what nginx does (http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_timeout)